purchasing/sherpa/views.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+from flask import (
+    Blueprint, render_template, url_for
+)
+from purchasing.extensions import login_manager
+from purchasing.users.models import User
+
+blueprint = Blueprint(
+    'sherpa', __name__, url_prefix='/sherpa',
+    static_folder='../static', template_folder='../templates'
+)
+
+@login_manager.user_loader
+def load_user(userid):
+    return User.get_by_id(int(userid))
+
+@blueprint.route("/", methods=["GET", "POST"])
+def index():
+    '''
+    Landing page for sherpa
+    '''
+    title = "Explore the Pittsburgh Procurement Process!"
+    pre_button_content = [{'title':'What are you doing?', 'content':'Need to start a contract or wondering which process you should use to buy something? Get started by selecting an option below!'}]
```

Comment:
Make sure you are conforming to [PEP 8](https://www.python.org/dev/peps/pep-0008/) standards about spacing, breaking up some of these lines on this stuff.